src/linterChecks/index.js

```diff
@@ -3,6 +3,11 @@ const { getLinterErrorCount } = require('./utils');
 
 module.exports = (testPath, tech) => {
   const eslintResponse = [];
+
+  if (tech === 'angular') {
+    return eslintResponse;
+  }
```

Comment:
```suggestion
  if (tech === 'angular') return eslintResponse;
```

Reply:
Current `eslint-config-wolox` enforce to use that way.